.github/workflows/go-test.yml

```diff
@@ -37,7 +35,7 @@ jobs:
       - name: Run linter
         if: always()
         run: |
-          go install honnef.co/go/tools/cmd/staticcheck@v0.3
+          go install honnef.co/go/tools/cmd/staticcheck@v0.4.0
```

Comment:
`0.4` means `0.4.x`. `0.4.0` is pinned to 0 as minor afaik

Reply:
Changed to `0.4`